packages/playground/src/registerServiceWorker.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
No idea why I was suddenly getting these errors, so I added the comments to save time since they blocked rendering of playground.